code/modules/food_and_drinks/machinery/smartfridge.dm

```diff
@@ -269,19 +269,6 @@
 	base_build_path = /obj/machinery/smartfridge/drying_rack //should really be seeing this without admin fuckery.
 	var/drying = FALSE
 
-/obj/machinery/smartfridge/drying_rack/Initialize(mapload)
-	. = ..()
-
-	// Cache the old_parts first, we'll delete it after we've changed component_parts to a new list.
-	// This stops handle_atom_del being called on every part when not necessary.
-	var/list/old_parts = component_parts.Copy()
-
-	component_parts = null
-	circuit = null
```

Comment:
Are we sure none of this is necessary? We don't want drying racks to drop *anything*.

I think you probably should've kept this as is but changed QDEL_LIST to:

```dm
for (var/obj/part_object in old_parts)
    qdel(part_object)
```

Reply:
I have no idea how this fridge works So yeah i'll do that

Reply:
But i don't understand why aren't we doing this for every machine then. Whatever i don't want that job anyway

Reply:
Hold on we don't need to do this because when finishing a machine frame with a screwdriver this piece of code

https://github.com/tgstation/tgstation/blob/be149d219c0ae9826837e35172b4ad5990166650/code/game/machinery/constructable_frame.dm#L228-L235

Will do that clean-up for us. So, it's unnecessary